packages/app/src/TeamPage.tsx

```diff
@@ -975,6 +1222,8 @@ export default function TeamPage() {
               <SourcesSection />
               <ConnectionsSection />
               <IntegrationsSection />
+              <TeamNameSection />
```

Comment:
I think maybe it makes sense to put the team name at the top, like in V1?

![image](https://github.com/user-attachments/assets/81e699a0-f501-4122-ba71-c4b18699bb42)

Reply:
(along with API keys)